File: iniciante/javascript/1154.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

let idade
let somaIdades = 0
let countIdades = 0

do{
    idade = parseFloat(lines.shift())
    if(idade >= 0){
        somaIdades += idade
        countIdades++ 
    } 
    
}while(idade > 0)

console.log(`${(somaIdades/countIdades).toFixed(2)}`)

File: iniciante/python/1154.py
# -*- coding: utf-8 -*-

soma_idades = 0
count_idades = 0
while True:
    entrada = int(input())
    if entrada < 0:
        break
    count_idades += 1
    soma_idades += entrada

print(f'{(soma_idades/count_idades):.2f}')
